documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

```diff
@@ -46,58 +48,7 @@ Please ensure you are not writing to any USB devices whilst the installer is run
 
 ==== Building `rpiboot` on your host system (Cygwin/Linux)
```

Comment:
Perhaps this should be Linux/Cygwin/WSL/macOS as that's what https://github.com/raspberrypi/usbboot/blob/master/Readme.md#building says? (or might just be easier to remove the part in brackets)

Reply:
Removed brackets, the currently supported platforms are best tracked with the source code.